test/cli/test_pass.rb

```diff
@@ -0,0 +1,166 @@
+# frozen_string_literal: true
+
+require "helpers"
+
+# tests cli command: pass
+class CLIPassTest < Minitest::Test
+  include Aruba::Api
+  include Helpers
+
+  def setup
+    setup_aruba
+  end
+
+  def test_pass
+    label = "test-login"
+    username = "user"
+    password = "pass"
+    output = password
+
+    # create login:
+    run_command "kbsecret new login -x #{label}" do |cmd|
+      cmd.stdin.puts "#{username}:#{password}"
+      cmd.stdin.close
+      cmd.wait
+    end
+
+    # retrieve login:
+    run_command "kbsecret pass #{label}" do |cmd|
+      cmd.wait
+      assert_equal output, cmd.output.chomp
+    end
+  ensure
+    # remove login:
+    run_command_and_stop "kbsecret rm #{label}"
+  end
+
+  def test_session
```

Comment:
`test_pass_with_session` would be a bit more descriptive.

Reply:
Got it. I didn't realize that you were naming the methods after options -- that makes a lot more sense. That's fine then!